vnext/Microsoft.ReactNative/LinkingHelper.cpp

```diff
@@ -0,0 +1,23 @@
+#include "pch.h"
```

Comment:
I'm not sure that this thing should be called LinkingHelper.  Maybe just LinkingModule?  Especially if we add OpenUri to it.

Should it be 

```
LinkingModule {
  static LinkingModule Get(propertybag);
  void SetInitialUrl(string);
  string GetIniitalUrl();
  void OpenUri(string);
};
```
Or keep all the methods static?  -- I feel like this is a good thing to be thinking about what we want the general pattern to look like.